usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -47,7 +48,7 @@ def universal_transaction_matview_annotations():
         ),
         "disaster_emergency_fund_codes_for_overall_award": Case(
             When(
-                action_date__gte="2020-04-01",
+                transaction__action_date__gte=datetime.date(2020, 4, 1),
```

Comment:
Are you seeing different results with using `transaction__action__date__gte` instead of `action_date__gte`? I should function the same since `action_date` on UTM is made up of the `transaction_normalized.action_date`.

Reply:
Yes, I'm not entirely sure as to why, but it does not correctly filter out pre `2020-04-01` dates unless we link to the transaction action date.